globalsearch.php

```diff
@@ -86,7 +86,6 @@
 								<input id='searchfor' name='searchfor' size='30' type='td' value=''>
 							</td>
 						</tr>
-						<tr style='font-size:0.5em;'>
```

Comment:
Only the `style` attribute should be removed, not the whole element. The `<tr>` is part of the table structure.